demos/tutorial/intro.php

```diff
@@ -171,24 +188,27 @@ protected function init(): void
     );
 });
 
-$wizard->addStep('Persistence', function ($page) {
+$wizard->addStep('Persistence', function ($page) use ($app) {
     $t = \atk4\ui\Text::addTo($page);
     $t->addParagraph(
         <<< 'EOF'
             Once your model is defined, it can be re-used later with any generic view:
             EOF
     );
 
-    Demo::addTo($page)->setCodeAndCall(function (View $owner) {
+    Demo::addTo($page)->setCodeAndCall(function (View $owner) use ($app) {
         session_start();
 
-        $model = new \atk4\ui\demo\DemoInvoice(new \atk4\data\Persistence\Array_($_SESSION['x'] ?? []));
+        $model = new \atk4\ui\demo\DemoInvoice(new \atk4\data\Persistence\Array_($_SESSION['x'] ?? []), ['dateFormat' => $app->ui_persistence->date_format]);
         $model->onHook(\atk4\data\Model::HOOK_AFTER_SAVE, function ($model) {
             $_SESSION['x'][$model->getId()] = $model->get();
         });
 
+        Header::addTo($owner, ['Record display in Card View using model data.']);
         $model->tryLoad(1);
-        \atk4\ui\Card::addTo($owner)->setModel($model, ['date']);
+        $model->loaded()
```

Comment:
use if/else, ternary should be used for passing a or b, but not for decisions, improve readability...

Reply:
ok